src/Microsoft.ML.Transforms/TensorflowTransform.cs

```diff
@@ -0,0 +1,374 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Transforms;
+using Microsoft.ML.Transforms.TensorFlow;
+
+[assembly: LoadableClass(TensorflowTransform.Summary, typeof(IDataTransform), typeof(TensorflowTransform.Arguments), typeof(SignatureDataTransform),
+    TensorflowTransform.UserName, TensorflowTransform.ShortName)]
+
+// This is for de-serialization from a binary model file.
+[assembly: LoadableClass(typeof(TensorflowTransform.TensorFlowMapper), null, typeof(SignatureLoadRowMapper),
+    "", TensorflowTransform.TensorFlowMapper.LoaderSignature)]
+
+[assembly: EntryPointModule(typeof(TensorflowTransform))]
+
+namespace Microsoft.ML.Transforms
+{
+    public static class TensorflowTransform
```

Comment:
>TensorflowTransform [](start = 24, length = 19)

We should decide whether we use "Tensorflow" or "TensorFlow" and be consistent.

It seems that TensorFlow is how it's written on www.tensorflow.org.

Reply:
Agreed, I've tried to standardize on captial T (indeed one place where I forgot caused a failure on linux due to case sensitivity)

---
In reply to: [212769559](https://github.com/dotnet/machinelearning/pull/704#discussion_r212769559) [](ancestors = 212769559)